mass_mailing_partner/models/mail_mass_mailing_contact.py

```diff
@@ -23,8 +23,10 @@ def _check_partner_id_list_ids(self):
                     ('id', '!=', contact.id)
                 ])
                 if contact.list_ids & other_contact.mapped('list_ids'):
-                    raise ValidationError(_("Partner already exists in one of "
-                                            "these mailing lists"))
+                    raise ValidationError(
+                        _("Partner already exists in one of "
+                          "these mailing lists", contact.partner_id.display_name)
```

Comment:
There's no ' %s', so this will give error. For not changing translations, you can do:
```suggestion
                          "these mailing lists") + ": %s" % contact.partner_id.display_name
```